### DISCUSSION TITLE
Rangeslider

### DISCUSSION THREAD

**Discussion by MamaTerror** [TomSchimansky/CustomTkinter#1368] [Q&A] [answered]
Hey! Does anybody has a solution for a rangeslider in ctk? Something like this in the picture below? 
![Bild1](https://user-images.githubusercontent.com/116916605/226481534-cb28b988-9ee8-41a2-8e2a-be5be52b0897.jpg)

**Comment by Akascape** [comment, marked as answer]
@MamaTerror Already requested: https://github.com/TomSchimansky/CustomTkinter/issues/118

There is a separate range slider module which we can use but that one is not dynamically resizable.

One guy also created a range slider in customtkinter but it is not compatible with the latest version. 
I will try to modify it in future if possible.

**Reply by MamaTerror** [reply]
Thanks a lot, im excited to see it :)

**Comment by Akascape** [comment]
Hi @MamaTerror,
I released this range slider for customtkinter as an addon: https://github.com/Akascape/CTkRangeSlider

![Screenshot](https://user-images.githubusercontent.com/89206401/229349732-0e078b00-f2b7-46e7-9774-313090253769.jpg)

**Reply by MamaTerror** [reply]
Hey Mate! Youre Perfect, ill try  later in my Code but i am thankfull in advance:) Made my day, hopefully! And thanks a lot
